Milliner/composer.json

```diff
@@ -31,7 +31,7 @@
         ],
         "test": [
             "@check",
-            "phpunit"
+	    "phpunit --coverage-html 'reports/clover_html' tests/"
```

Comment:
clover_html might not be correct here - it is the case on my machine, but is that the case in general?

Reply:
Yeah codecov uses the XML output (clover.xml). So you can remove this. I normally add it when I am testing on my machine and then remove before committing as it will just generate a directory of un-used HTML coverage reports.

Reply:
Thanks - I'll take this out.